src/theme/components/customTable.ts

```diff
@@ -0,0 +1,29 @@
+const baseStyle = ({ showRowDividers, columnHeadersTextColor }) => ({
+  th: {
```

Comment:
Add a style to set the letter spacing to 0.  You did that for `td`, but it also needs to be added here.
```
letterSpacing: "0",
```